guides/get-started-microservices-on-kubernetes/simple-microservice-database-part1.adoc

```diff
@@ -0,0 +1,413 @@
+
+* install the JDBC Driver for the PostgreSQL Database into WildFly
+* add a database connection to PostgreSQL in WildFly
+
+=== pom.xml
+
+==== dependencies
+
+Add the following dependencies to the `pom-xml` file `dependencies` section:
+
+[source,xml,subs="normal"]
+----
+        <dependency>
+            <groupId>jakarta.persistence</groupId>
+            <artifactId>jakarta.persistence-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.transaction</groupId>
+            <artifactId>jakarta.transaction-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+----
+
+==== wildfly-maven-plugin
+
+Add the `wildfly-datasources-galleon-pack` feature-pack and the `postgresql-datasource` layer to the `wildfly-maven-plugin` configuration;
+
+You should end up with the link:https://github.com/wildfly/wildfly-maven-plugin/[wildfly-maven-plugin] configured like in the following:
+
+[source,xml,subs="normal"]
+----
+    <plugin>
+        <groupId>org.wildfly.plugins</groupId>
+        <artifactId>wildfly-maven-plugin</artifactId>
+        <version>{version-wildfly-maven-plugin}</version>
+        <configuration>
+            <feature-packs>
+                <feature-pack>
+                    <location>org.wildfly:wildfly-galleon-pack:{version-wildfly-galleon-pack}</location>
+                </feature-pack>
+                <feature-pack>
+                    <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:{version-wildfly-cloud-galleon-pack}</location>
+                </feature-pack>
+                <feature-pack>
+                    <location>org.wildfly:wildfly-datasources-galleon-pack:{version-wildfly-datasources-galleon-pack}</location>
+                </feature-pack>
+            </feature-packs>
+            <layers>
+                <layer>cloud-server</layer>
+                <layer>postgresql-datasource</layer>
+            </layers>
+        </configuration>
+        <executions>
+            <execution>
+                <goals>
+                    <goal>package</goal>
+                </goals>
+            </execution>
+        </executions>
+    </plugin>
+----
+
+=== Java Classes
+
+Add the following classes to the project:
+
+.org.wildfly.examples.model.TestTable   :
+[source,java]
+----
+package org.wildfly.examples.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.xml.bind.annotation.XmlElement;
+import jakarta.xml.bind.annotation.XmlRootElement;
+
+import java.util.Objects;
+
+import static jakarta.persistence.GenerationType.IDENTITY;
+
+@SuppressWarnings("serial")
```

Comment:
This looks like it is a redundant supression

Reply:
Thank you: removed!